plan_test.go

```diff
@@ -357,32 +357,6 @@ func TestQueryPlanFragmentSpread1(t *testing.T) {
 	PlanTestFixture1.Check(t, query, plan)
 }
 
-func TestQueryPlanFragmentSpread1DontDuplicateTypename(t *testing.T) {
```

Comment:
This test is no longer applicable now that `__typename` is, by design, always appended and allowed to repeat a user-defined selection.